examples/components/http-password-checker/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/http-password-checker.js",
   "type": "module",
   "scripts": {
-    "generate:types": "jco types wit/ -o generated/types",
+    "generate:types": "rimraf generated/types && jco types wit/ -o generated/types",
```

Comment:
Since the types are required for the build, can we add the `generate:types` script as a post-install script?

```jsonc
"scripts": {
  // ...
  "postinstall": "npm run generate:types",
  // ...
}
```